templates/projects/contact.html

```diff
@@ -0,0 +1,134 @@
+{% macro render(owner, current_user, project, n_tasks, num_expected_task_runs, num_remaining_task_runs, n_available_tasks_for_user, n_completed_tasks_by_user) %}
+<div id="contact-container">
+    <!-- Modal -->
+    <div class="modal" id="contact-modal" tabindex="-1" role="dialog" aria-labelledby="contact-modal-label" aria-hidden="true">
+        <div class="modal-dialog" role="document">
+        <div class="modal-content">
+          <div class="modal-header">
+            <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+              <span aria-hidden="true">&times;</span>
+            </button>
+            <h4 class="modal-title" id="contact-modal-label">Contact Us About: {{ project.name }}</h4>
+          </div>
+          <div class="modal-body">
+            <div class="row header">
+                <div class="col col-md-1">
+                    <span class="stepwizard-step">
+                        <span class="btn btn-circle btn-lg btn-border step-done"><i aria-hidden="false" class="fa fa-envelope-o icon-white"></i></span>
+                    </span>
+                </div>
+                <div class="col col-md-11 text">
+                    <div class="open-text">
+                        Please include your message below and we'll contact you shortly. Thank you.
+                    </div>
+                    <div class="close-text hide">
+                        Thank you. Your message has been sent.
+                    </div>
+                </div>
+            </div>
+            <div class="row body">
+                <div class="col col-md-12">
+                <form>
+                    <div class="form-group">
+                        <label for="txtEmail">Email address</label>
+                        <input type="email" class="form-control" id="txtEmail" readonly aria-describedby="emailHelp" placeholder="Enter email" value="{{ current_user.email_addr }}">
+                    </div>
+                    <div class="form-group">
+                        <label for="message">Message</label>
+                        <textarea class="form-control" id="txtMessage" rows="3"></textarea>
+                    </div>
+                </form>
+            </div>
+            </div>
+          </div>
+          <div class="modal-footer">
+            <div class="error text-danger pull-left hide">There was a problem sending your message. Please try again.</div>
+            <button type="button" class="btn btn-open btn-secondary" data-dismiss="modal">Cancel</button>
+            <button type="button" class="btn btn-open btn-primary">Send</button>
+            <button type="button" class="btn btn-close btn-light hide" data-dismiss="modal">OK</button>
+          </div>
+        </div>
+    </div>
+</div>
+<script type="text/javascript">
+    (function(){
+        $('#contact-container .btn-primary').click(function() {
```

Comment:
Just out of curious , is there any reason to pass data to the endpoint instead of generating data in `pybossa/view/projects.py`?

Reply:
Just for convenience since the values were present in the view.